CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ All notable changes to this project will be documented in this file.
     * #### Added
     * #### Changed
     * #### Removed
+        * `easy_install` support dropped.
```

Comment:
FWIW, this is under "Removed" which makes it redundante to say "dropped" in the line item.